### PR TITLE
WIP - MVAPICH2-2.1-GNU-5.1.0-2.25

### PR DESCRIPTION
**Pull request by jas02**
Error during configure step:

```
ERROR: Build of /easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.1-GNU-5.1.0-2.25.eb failed (err: 'build failed (first 300 chars): Both $F90 and $FC set, can I overwrite $FC with $F90 (gfortran) ?')
```

### CODE REVIEW

File: easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.1-GNU-5.1.0-2.25.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+homepage = 'http://mvapich.cse.ohio-state.edu/overview/mvapich2/'
+description = "This is an MPI 3.0 implementation.  It is based on MPICH2 and MVICH."
+
+toolchain = {'name': 'GNU', 'version': '5.1.0-2.25'}
+
+source_urls = ['http://mvapich.cse.ohio-state.edu/download/mvapich/mv2/']
+sources = [SOURCELOWER_TAR_GZ]
+
+preconfigopts = 'unset F90 && unset FC && unset F90FLAGS && unset FCFLAGS && '
```
Comment (boegel) [inline easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.1-GNU-5.1.0-2.25.eb]: this is futile, the MVAPICH2 easyblock checks for `$FC` and `$F90`, prior to even looking at `(pre)configopts`

something is setting `$FC`, while `eb` itself defines `$F90` via the toolchain support

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
missing deps:

```
ERROR: Irresolvable dependencies encountered: Bison/2.7-GNU-5.1.0-2.25
```

**Comment by migueldiascosta** [comment]
@jas02, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**